aws-inventory/lambda/inventory-elasticbeanstalk.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019-2020 Turner Broadcasting Inc. / WarnerMedia
+# Copyright 2021 Chris Farris <chrisf@primeharbor.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+from botocore.exceptions import ClientError
+import json
+import os
+import time
+from datetime import datetime, timezone
+from dateutil import tz
+
+from antiope.aws_account import *
+from common import *
+
+import logging
+logger = logging.getLogger()
+logger.setLevel(getattr(logging, os.getenv('LOG_LEVEL', default='INFO')))
+logging.getLogger('botocore').setLevel(logging.WARNING)
+logging.getLogger('boto3').setLevel(logging.WARNING)
+logging.getLogger('urllib3').setLevel(logging.WARNING)
+
+RESOURCE_PATH = "eb/application"
+TYPE = "AWS::ElasticBeanstalk::Application"
```

Comment:
You want to create two here one for Environments and one for Applications. The different things should get different types.

Reply:
Same with RESOURCE_PATH. Create one for each of the eb/application and eb/environments